avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_1/fruits_day1

```diff
@@ -1,6 +1,7 @@
-mon_fruits,apple
 mon_fruits,pear
+mon_fruits,apple
 mon_fruits,banana
 mon_fruits,kiwi
+mon_fruits,apple,123
 mon_fruits,grapes
-mon_fruits,pineapple
+mon_fruits,pineapple
```

Comment:
interesting as there is no change but git is showing adding/deleting lines here.